libs/design/src/atoms/form/form-field/form-field.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { DaffFormFieldComponent } from './form-field.component';
+
+describe('DaffFormFieldComponent', () => {
+  let component: DaffFormFieldComponent;
+  let fixture: ComponentFixture<DaffFormFieldComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ 
+        DaffFormFieldComponent
+      ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(DaffFormFieldComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
+
```

Comment:
I don't think these belong in unit testing. Also, the form-field component doesn't know anything about `daff-input` or `daff-select`. It just knows to place a `daff-error-message` in a particular place. And I think that would make more sense in a screenshot test or something.

Reply:
@lderrickable, I'm not saying that we should test how it looks, but I think we should test which elements it can/can't transclude, as well as how they are transcluded.

Reply:
I think that to test that, I'd need to add a wrapping element around the 
```
<ng-content select="daff-error-message"></ng-content>
```
because you can't set a class on an ng-content. Is there another way? And when it comes down to it, I still think this is testing where the `daff-error-message` is positioned in relation to other transcluded content.

Reply:
actually nevermind, I think i need to add the wrapping class for styling changes anyway, so it'll already be there

Reply:
done

Reply:
So, I removed the test, because I no longer have the wrapper class for `.daff-error-message`s

Reply:
@lderrickable can you write a test that asserts that you can't transclude random content? For example, I'd like to assert that I CAN transclude the following

```html
<daff-form-field>
  <input daffInput />
  <daff-error-message></daff-error-message>
</daff-form-field>
```

But not,

```html
<daff-form-field>
  <input daffInput />
  <p>Some Random Paragraph tag</p>
</daff-form-field>
```

Reply:
So I couldn't do the ContentChild thing. Turns out the way material does it is they actually transclude everything with an `ng-content`. So to only transclude certain things, I think I need to do 
```
<ng-content select="[daff-input]"></ng-content>
etc...
```
I made that change